### PR TITLE
Posts in events

### PR DESCRIPTION
**Pull request by nilsen340**
Given the fact that we provide posts with the event endpoint we should map them into the events returned by the SDK. 

@onurakpolat @xla could you please do a quick review?

### CODE REVIEW

File: Classes/TGEventManager.m [hunk 1]
```diff
@@ -504,11 +504,20 @@ - (void)handleSingleEventResponse:(NSDictionary*)jsonResponse withError:(NSError
...
+    for(NSDictionary *postData in postDictionaries) {
+        [TGPost createOrLoadWithDictionary:postDictionaries[postData]];
+    }
+    
     NSArray *eventDictionaries = [jsonResponse objectForKey:@"events"];
     NSMutableArray *events = [NSMutableArray arrayWithCapacity:eventDictionaries.count];
 
     for (NSDictionary *eventData in eventDictionaries) {
         TGEvent *newEvent = [TGEvent createOrLoadWithDictionary:eventData];
+        if([newEvent.type  isEqual: @"tg_like"]) {
```
Comment (onurakpolat) [inline Classes/TGEventManager.m]: We should start using constants more often for things like `events` or `tg__like` but ok for now.

**Comment by onurakpolat** [comment]
@nilsen340 please more context into description next time ;)

Other than that :+1:

File: Classes/TGEventManager.m (hunk 1 above)
Comment (nilsen340) [inline Classes/TGEventManager.m]: actually, yes. Ill add that.

**Comment by nilsen340** [comment]
@onurakpolat I've fixed what we discussed earlier and also added user parsing on the event fetch. This is what made events sometimes miss users, as if events were fetched before users there would be no user to add to the event object. Feel free to have another look at it.

**Comment by onurakpolat** [comment]
:+1:

**Pull request merged**